lib/faulty/circuit.rb

```diff
@@ -156,6 +156,15 @@ def finalize
       end
     end
 
+    # @return [String] Text representation of the circuit
+    def inspect
+      # rubocop:disable Layout/LineLength
+      interested_opts = %i[cache_expires_in cache_refreshes_after cache_refresh_jitter cool_down evaluation_window rate_threshold sample_threshold errors exclude]
+      # rubocop:enable Layout/LineLength
```

Comment:
Let's stick to the style guide when we can instead of circumventing it.

```suggestion
      interested_opts = %i[
        cache_expires_in
        cache_refreshes_after
        cache_refresh_jitter
        cool_down evaluation_window
        rate_threshold
        sample_threshold
        errors exclude
      ]
```